operators/CrunchyData_postgres-operator/alarm-inspection-report.md

```diff
@@ -0,0 +1,17 @@
+# True Alarm Report
```

Comment:
I left the analysis for less interesting alarms (false alarms, acto misoperation and slow kind cluster ) as additional columns in the result.csv file.